nuxeo-java-client/src/main/java/org/nuxeo/client/objects/Repository.java

```diff
@@ -192,11 +203,21 @@ public void query(String query, Callback<Documents> callback) {
         fetchResponse(api.query(query), callback);
     }
 
+    /**
+     * @deprecated since 3.7
+     */
+    @Deprecated
     public void query(String query, String pageSize, String currentPageIndex, String maxResults, String sortBy,
             String sortOrder, String queryParams, Callback<Documents> callback) {
         fetchResponse(api.query(query, pageSize, currentPageIndex, maxResults, sortBy, sortOrder, queryParams),
                 callback);
     }
```

Comment:
Why not redirect the call as you did here: https://github.com/nuxeo/nuxeo-java-client/pull/88/files#diff-3df9f9c5f8df7db978f549552a30797bR185 ?

```suggestion
    @Deprecated
    public void query(String query, String pageSize, String currentPageIndex, String maxResults, String sortBy,
            String sortOrder, String queryParams, Callback<Documents> callback) {
        query(query, pageSize, currentPageIndex, maxResults, sortBy, sortOrder, new String[] { queryParams }, callback);
    }
```
so all 4 methods would call https://github.com/nuxeo/nuxeo-java-client/pull/88/files#diff-8074d8a2dc187eecd970c22a69e24072R114-R119 ?

Reply:
You're right, thanks 👍